src/Serialization/main.jl

```diff
@@ -53,8 +53,13 @@ const typeMap = Dict{Type, String}([
     Nemo.GaloisField => "Nemo.GaloisField",
     Nemo.GaloisFmpzField => "Nemo.GaloisFmpzField",
     fmpz => "fmpz",
+    fmpq => "fmpq",
     gfp_elem => "Nemo.gfp_elem",
     gfp_fmpz_elem => "Nemo.gfp_fmpz_elem",
+    fmpq_mpoly => "fmpq_mpoly",
+    fmpz_mpoly => "fmpz_mpoly",
+    Nemo.NmodRing => "Nemo.NmodRing",
+    Hecke.NfRel{nf_elem} => "Hecke.NfRel{nf_elem}",
```

Comment:
Meta-remark: While adding these entries is fine, in reality right now doing this resp. the whole `typeMap` is almost completely irrelevant: e.g. it contains some "unused" entries (`Vector`...) which one doesn't notice, and several types are missing and added during runtime via the generic fallback code involving `eval` later on. As long as we do that, there' no point in touching this at all. I'll make a PR with a proposal for changing that.